### PR TITLE
Suggested edits to documentation text

### PR DESCRIPTION
**Pull request by DougManuel**
The following changes were made. See below for requested additions from @zargot and @yulric

1. Tutorials were renamed to be more descriptive:
- Tutorial 1 —> Getting started
- Tutorial 2 —> Getting to know the validation library

Edits were made throughout the tutorials. No change in content:
- minor rearrangements in the content. 
- Headings for tutorial 2.
- Grammar and spelling. 

2. Added sections to the index page. The added sections are often found in Python library documentation. A better option may be to put the ‘usage’ section on a separate page. 
- Overview section. This section seems straightforward to add.
- Installation. I recommend we add an installation section. Yes, there is an install section in one of the tutorials, but users expect installation on the main page. I suggest that we keep installation in both the index and tutorial. If we want to install instructions in one place, it would be the main index page.
- Usage. Again this section is somewhat duplicative of the tutorial and explanation guide. These usage sections would be 'how-tos'. Short and to the point.
- FAQ. I suggest that we point people to Discourse and GH Issues. This section seems duplicative of the ‘contributing’ section, but they are different concepts. Thoughts? We would need to add a few FAQs to the Discourse forum. 

@zargot, could you add the missing pieces to the index page?

### CODE REVIEW

**Comment by DougManuel** [comment]
@zargot could you check the links.  I inserted placeholders for a few links. Could you search:
- “insert the link to the missing_mandatory_column rule”
- “insert the link to data type rule”

File: docs/examples/explanation-document/explanation-document.qmd [hunk 1]
```diff
@@ -41,35 +48,34 @@ def pprintDictList(dictList, title):
     console.print(table)
 ```
 
-The library has two main functions described below:
-
-1. `validate_data`: which is used to validate an ODM dataset using a validation schema
-2. `generate_validation_schema`: which is used to generate a validation schema from the ODM dictionary
-
-The next chunk will import these functions.
+Next, install the two main library functions.
```
Comment (yulric) [inline docs/examples/explanation-document/explanation-document.qmd:51]: Next, **import** the two main library functions.

File: docs/examples/explanation-document/explanation-document.qmd [hunk 2]
```diff
@@ -41,35 +48,34 @@ def pprintDictList(dictList, title):
...
+* `schema`: which has the validation rules
 
-The structure of the validation rules follows a Python library called [cerberus](https://docs.python-cerberus.org/en/stable/schemas.html). Cerberus does all the validation heavy lifting. The PHES-ODM validation package wraps around cerberus to integrate the ODM dataset and the ODM schema into cereberus.
+The structure of the validation rules follows a Python library called [cerberus](https://docs.python-cerberus.org/en/stable/schemas.html) that does all the validation heavy lifting. The PHES-ODM validation package integrates the ODM schema into cereberus and then uses cereberus methods to validate ODM data. 
 
-The dataset in the next chunk comes from a [CSV file](./datasets/invalid-sites-dataset.csv) and represents a `sites` table. The dataset is invalid since its missing its `geoLat` column.
+The dataset in the next code chunk is a very simple `sites` table as a [CSV file](./datasets/invalid-sites-dataset.csv). The dataset has invalid data since it is missing the mandatory `geoLat` column. You can see in the above schema that `geolat` is mandatory by code `'geoLat': {'required': True}`.
```
Comment (yulric) [inline docs/examples/explanation-document/explanation-document.qmd:78]: `geolat` -> `geoLat`

File: docs/examples/explanation-document/explanation-document.qmd [hunk 3]
```diff
@@ -91,13 +103,17 @@ validation_result = validate_data(
...
 
-Each field in the error report object provides metadata to trace where the error came from. The `message` field gives a human readable description of the error. Although most of the fields are self-explanatory, if further clarification is needed, the `errorType` field can be used to dig deeper by finding the specification file for the validation rule it refers to. For example the `missing_mandatory_column` specification can be found in the [repo](../../validation-rules/missing_mandatory_column.md).
+The validation report also consists of versioning metadata fields to debug any errors in the ODM schema. These are the `data_version`, `schema_version`, and `package_version` fields.
 
-The validation report also consists of versioning metadata fields to help us debug any package errors. These are the `data_version`, `schema_version`, and `package_version` fields.
+## Example: validating a validate data table
```
Comment (yulric) [inline docs/examples/explanation-document/explanation-document.qmd:114]: Should be a third level header? ## -> ###

File: docs/examples/explanation-document/explanation-document.qmd [hunk 4]
```diff
@@ -117,41 +133,44 @@ pprint(
 
 As we can see, the error list is empty.
 
-Creating your own validation schemas can get tedious due to the breadth of information in the dictionary. With the `generate_validation_schema` function, we can generate a validation schema from the ODM dictionary. The next chunk has a snippet from the [parts sheet](./dictionary/parts-v2.csv) in the dictionary.
+### `generate_validation_schema` function
+
+Generally, you don't need to create a validation schema because there are default OMD validation schemas for all ODM versions. However, you can create your own schema with the `generate_validation_schema` function. This function generates a validation schema from the ODM dictionary. The next chunk has a chunk from the [parts sheet](./dictionary/parts-v2.csv) in the dictionary.
```
Comment (yulric) [inline docs/examples/explanation-document/explanation-document.qmd:138]: OMD -> ODM?

**Review by yulric — CHANGES_REQUESTED** [review]
Do we need the Jupyter notebook?

**Comment by zargot** [comment]
> Do we need the Jupyter notebook?

I removed it :)

**Comment by DougManuel** [comment]
> > Do we need the Jupyter notebook?
> 
> I removed it :)

Jupyter files may have been generated during quarto render, but that should only be an intermediate file. Right?

File: docs/examples/explanation-document/explanation-document.qmd (hunk 1 above)
Comment (zargot) [reply docs/examples/explanation-document/explanation-document.qmd]: done

File: docs/examples/explanation-document/explanation-document.qmd (hunk 2 above)
Comment (zargot) [reply docs/examples/explanation-document/explanation-document.qmd]: done

File: docs/examples/explanation-document/explanation-document.qmd (hunk 3 above)
Comment (zargot) [reply docs/examples/explanation-document/explanation-document.qmd]: done

File: docs/examples/explanation-document/explanation-document.qmd (hunk 4 above)
Comment (zargot) [reply docs/examples/explanation-document/explanation-document.qmd]: done

**Comment by zargot** [comment]
> > > Do we need the Jupyter notebook?
> > 
> > 
> > I removed it :)
> 
> Jupyter files may have been generated during quarto render, but that should only be an intermediate file. Right?

Yes, that's correct. I added them to the gitignore now. :)

**Review by yulric — APPROVED** [review]

**Pull request merged**